src/pages/contact/inez.astro

```diff
@@ -1,18 +0,0 @@
----
```

Comment:
This actually might conflict with Refactor 1. I made some changes to these files, but now deleting. Also the `contact/index.astro` page has more significant changes now. Whichever one merges, just take all changes from this PR.